synapse/storage/event_push_actions.py

```diff
@@ -36,7 +36,8 @@ def __init__(self, hs):
             self.EPA_HIGHLIGHT_INDEX,
             index_name="event_push_actions_u_highlight",
             table="event_push_actions",
-            columns=["user_id", "stream_ordering", "highlight"],
+            columns=["user_id", "stream_ordering"],
+            where_clause="highlight = 1",
```

Comment:
Are you sure about this? I thought it was possible to query this without the "highlight = 1" clause.